docs/installation.rst

```diff
@@ -119,11 +119,14 @@ Make sure you have a valid python 3.6+ installation.
 See :ref:`python` if you don't know how.
 
 1. Download the latest source code from `here <https://github.com/Kruptein/PlanarAlly/releases/>`_.
-2. Open the server folder.
-3. Execute `pip install -r requirements.txt`.
-4. Run the server by executing `python planarserver.py`.
-5. Optionally you can configure the server in the `server_config.cfg` file.
-6. Restart the server after applying a change.
+2. Open the client folder.
+3. Execute `npm install`.
+4. Execute `npm run build`.
+5. Return to the PlanarAlly folder and open the server folder.
+6. Execute `pip install -r requirements.txt`.
```

Comment:
We also need to instruct users to install PA in a virtualenv - this way their systems will be left intact.

Reply:
`````markdown
5. Optionally create a new virtualenv to keep your system installation intact:
```
python3 -m venv .planarally_venv
```
6. If you created a virtualenv, you need to activate it: `source .planarally_venv`
`````